website/www/site/i18n/home/hero/en.yaml

```diff
@@ -0,0 +1,20 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- id: home-hero-title
+  translation: Introducing Apache Beam
+- id: home-hero-heading
+  translation: An advanced unified programming model
+- id: home-hero-subheading
+  translation: Implement batch and streaming data processing jobs that run on any execution engine.
+- id: home-hero-button
+  translation: Install Beam
```

Comment:
This hero header doesn't seem to be rendering correctly on mobile

![image](https://user-images.githubusercontent.com/675055/106960386-da903480-66f0-11eb-9063-6a3101d37772.png)

Reply:
The footer with the quickstarts looks off too:
![image](https://user-images.githubusercontent.com/675055/106960516-0ca19680-66f1-11eb-9ff6-ddf38acb7784.png)

Reply:
This seems to be resolved now.